src/shell/notifications.test.js

```diff
@@ -0,0 +1,88 @@
+import { screen } from '@testing-library/react'
+import { within } from '@testing-library/dom'
+import userEvent from '@testing-library/user-event'
+
+import { endDay, getItemByName } from '../test-utils/ui'
+import { STORM_MESSAGE } from '../strings'
+import { farmhandStub } from '../test-utils/stubs/farmhandStub'
+import { saveFileStubFactory } from '../test-utils/stubs/saveFileStubFactory'
+
+describe('notifications', () => {
+  test('pending notifications are shown when a day starts', async () => {
+    jest.spyOn(Math, 'random').mockReturnValue(0)
+
+    await farmhandStub()
+    await endDay()
+
+    // STORM_MESSAGE message is a sample pending day notification that is known
+    // to show upon starting a new day when Math.random() is locked to 0.
+    await screen.findByText(STORM_MESSAGE)
+  })
```

Comment:
no `expect` in these tests either

Reply:
Added in https://github.com/jeremyckahn/farmhand/commit/37e407d8117b4993d2e17cf968c54ee9cc6504aa!